tests/unit/test_zigzag.py

```diff
@@ -17,12 +17,12 @@
 # Globals
 # ======================================================================================================================
 # Shared expectations
-SHARED_TEST_LOG_EXP = {'build_url': 'BUILD_URL',
-                       'build_number': 'BUILD_NUMBER',
-                       'module_names': ['RPC_RELEASE',
-                                        'JOB_NAME',
-                                        'MOLECULE_TEST_REPO',
-                                        'MOLECULE_SCENARIO_NAME',
+SHARED_TEST_LOG_EXP = {'build_url': 'https://rpc.jenkins.cit.rackspace.net/job/PM_rpc-openstack-pike-rc-xenial_mnaio_no_artifacts-swift-system/78/',  # noqa
```

Comment:
Can you comment on the reason for changing these values?

Reply:
These values where updated in the tests because just having a string with any value is not acceptable to use with this feature.  The feature depends on the values that exist inside the strings.  

@johnduarte Would a comment in the commit be acceptable?

Reply:
I just pushed an update, please let me know if I have not adequately described whats going on.

Reply:
👍 works for me.